src/components/StripCard/StripCard.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { stripCardDriverFactory } from './StripCard.driver';
+import { StripCard } from './';
+import { stripCardTestkitFactory } from '../../testkit';
+import { stripCardTestkitFactory as enzymeStripCardTestkitFactory } from '../../testkit/enzyme';
+
+describe('StripCard', () => {
+  const createDriver = createUniDriverFactory(stripCardDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<StripCard />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  it('should render info', async () => {
+    const valueAsString = '<div>info!</div>';
+    const value = <div dangerouslySetInnerHTML={{ __html: valueAsString }} />;
```

Comment:
Why use `dangerouslySetInnerHTML`? I'm not sure I understand

Reply:
Shlomi didn't want us to have innerHTML in the driver and it was the only solution we found
https://github.com/wix/wix-ui-tpa/pull/63/files#diff-f3d7262b9300d4ce27455808474b8da2L27